.babelrc

```diff
@@ -1,4 +1,4 @@
 {
-  "presets": ["es2015", "react"],
-  "plugins": ["transform-object-rest-spread"]
+  "presets": ["cf"],
```

Comment:
Well then you should re-specify all the config inside the existing one: https://github.com/cloudflare/babel-preset-cf/blob/master/index.js#L2-L24

There's a good reason for every single line in that file. I offered to go over it all when I was still at cf and no one took me up on that.